.github/workflows/python-app.yml

```diff
@@ -27,6 +27,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install flake8 pytest pytest-cov
+        pip install .
```

Comment:
Do we actually need this?

Reply:
This is required as it pip installs the dependencies we have for the package in the setup.py

Reply:
Oh the dependencies are aren't in a requirements.txt and we're actually using them now